docker/Dockerfile

```diff
@@ -81,7 +65,11 @@ RUN mkdir /var/farmOS \
   && rm -r /opt/drupal && cp -rp /var/farmOS /opt/drupal
 
 # Set the entrypoint.
-COPY docker-entrypoint.sh /usr/local/bin/
-RUN chmod u+x /usr/local/bin/docker-entrypoint.sh
```

Comment:
Changing the permissions on the file, and persisting that into git helps to reduce this extra layer on the docker image